utils/pom.xml

```diff
@@ -37,6 +37,13 @@
             <version>18.0</version>
         </dependency>
 
+        <!-- JTar - Java Tar file support -->
+        <dependency>
+            <groupId>org.kamranzafar</groupId>
+            <artifactId>jtar</artifactId>
+            <version>2.3</version>
+        </dependency>
```

Comment:
Is this library a good for the purpose? never heard about it ;)

Reply:
Good catch, fixed!

Reply:
Yes, it is handy if you want to bundle a bunch of files up into a single archive, but you don't want to spend the CPU on compressing them. I could potentially have used the zip files for a similar purpose, but tar seems like a more suitable format for a simple concatenation of files.

It also leaves open the possibility of streaming them in the future, which isn't possible with zip.